src/Spryker/Install/Executable/CommandLine/CommandLineExecutable.php

```diff
@@ -43,7 +43,13 @@ public function __construct(CommandInterface $command, ConfigurationInterface $c
      */
     public function execute(StyleInterface $output): int
     {
-        $process = new Process($this->command->getExecutable(), SPRYKER_ROOT, getenv(), null, 600);
+        $process = new Process(
+            $this->command->getExecutable(),
+            SPRYKER_ROOT,
+            getenv(),
+            null,
+            $this->command->hasTimeout() ? $this->command->getTimeout() : 600
```

Comment:
Should we use 0 as default then as the 0 default suggests?
Or should be at least make this a constant above as TIMEOUT_DEFAULT or sth?